bin/tictactoe

```diff
@@ -1,3 +1,27 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic Tac Toe!"
+
+puts "What kind of Game would you like to play? O, 1, or 2 player."
+
+input = gets.strip
```

Comment:
I would make the input an Integer so that you do not have to worry about Uppercase or Lowercase inputs

```ruby 
input = gets.strip.to_i
```